src/main/java/com/vdovich/tbfm/util/JsonObjects.java

```diff
@@ -0,0 +1,14 @@
+package com.vdovich.tbfm.util;/* Created by user on 15.01.20 */
+
+public enum JsonObjects {
```

Comment:
Actually it's not json objects. It would be better to rename this class to something like JsonProperty or JsonField. Also, it's bad practice to name class in plural form.